### DISCUSSION TITLE
Pass-through events

### DISCUSSION THREAD

**Discussion by ElfWitch** [rust-windowing/winit#2559] [Q&A] [answered]
Is it possible to allow for events to pass through the window as if it wasn't there? I want to make a program similar to pureref which has a "transparent to mouse" mode. Preferably I would like keyboard and mouse events to be ignored by the window, effectively making it just an overlay.

**Comment by maroider** [comment, marked as answer]
This has been possible since Winit 0.27.0, with the addition of [`Window::set_cursor_hittest`](https://docs.rs/winit/latest/winit/window/struct.Window.html#method.set_cursor_hittest).

**Reply by ElfWitch** [reply]
Thank you so much! I missed that in the docs.
